tools/ddspreviewer/TexturePreview.qml

```diff
@@ -0,0 +1,109 @@
+import Qt3D.Core 2.12
+import Qt3D.Render 2.12
+import Qt3D.Input 2.12
+import Qt3D.Extras 2.12
+import QtQml 2.12
+import QtQuick.Controls.Material 2.4 as QQ2Material
+import Kuesa 1.0 as KUE
+
+Entity {
+    id: root3D
+    property string filePath
+    property int mode: _PLANE_MODE
+    readonly property int _PLANE_MODE: 1
+    readonly property int _SKYBOX_MODE: 2
+
+    onModeChanged: {
+        camera.position = Qt.vector3d(0, 0, -5)
+        camera.upVector = Qt.vector3d(0, 1, 0)
+        camera.viewCenter = Qt.vector3d(0, 0, 0)
+    }
+
+    components: [
+        RenderSettings {
+            activeFrameGraph: ForwardRenderer {
+                camera: camera
+                clearColor: QQ2Material.Material.color(QQ2Material.Material.Grey)
+            }
+        },
+        InputSettings {}
+    ]
+
+    TextureLoader {
+        id: ddsTexture
+        source: filePath
+    }
+
+    Camera {
+        id: camera
+        fieldOfView: 40
+        position: Qt.vector3d(0, 0, -5)
+        upVector: Qt.vector3d(0, 1, 0)
+        viewCenter: Qt.vector3d(0, 0, 0)
+    }
+
+    OrbitCameraController {
```

Comment:
Shouldn't this be disabled in plane mode? Or there's a use I don't see of moving the camera around in that mode?

Reply:
you probably still want to zoom in out in some cases